hazelcast/src/main/java/com/hazelcast/core/MemberLeftException.java

```diff
@@ -56,24 +56,26 @@ public Member getMember() {
     private void writeObject(ObjectOutputStream out) throws IOException {
         out.defaultWriteObject();
 
-        Address address = member.getAddress();
-        String host  = address.getHost();
-        int port = address.getPort();
-
-        out.writeUTF(member.getUuid());
-        out.writeUTF(host);
-        out.writeInt(port);
-        out.writeBoolean(member.isLiteMember());
+        if (member == null) {
```

Comment:
I don't understand this part. is this another bug this PR is fixing? NPE waiting to happen?